html/pages/device/overview/generic/sensor.inc.php

```diff
@@ -6,12 +6,60 @@
 }
 
 if (count($sensors)) {
+    switch (strtolower($sensor_type)) {
```

Comment:
Might want to put this into a function.  There are other places where sensor icons are used, such as the about page.